API/chat/views.py

```diff
@@ -96,3 +96,34 @@ def get(self, request, *args, **kwargs):
             json.dumps(channel),
             content_type='application/json'
         )
+"""
```

Comment:
Since this view is commented out, and there are no tests for it, why is it on this pull request?